app/typeform/pop-over-full/page.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { PopOver } from '@/components/TypeForm/PopOver';
+
+const TypeFormPage = () => {
```

Comment:
That was smart earlier that you separated the storyblok pages into a different storyblok route - thanks @sherakama !